frontend/apps/app/components/AppBar/AppBar.module.css

```diff
@@ -61,9 +61,8 @@
   position: relative;
 }
 
-.breadcrumbTrigger {
-  all: unset;
-  cursor: pointer;
+.breadcrumbItem:focus {
+  outline: none;
```

Comment:
This was added to prevent the blue border from appearing when clicking.

![ss 3186](https://github.com/user-attachments/assets/357fbdb3-01b1-4f26-a81c-d180c2af97be)

Reply:
I see! Thank you very much.
I added this CSS to try to make it look the way it did before the change.
I will modify it to use the brand colors!

Reply:
I fixed it!
https://github.com/liam-hq/liam/pull/1473/commits/37b8bd025211eb437993c0f5f901b0f668cbba6f

![ss 3187](https://github.com/user-attachments/assets/c0971485-6ac1-4e03-9a02-6a0c120333e0)